tests/foreman/data/hammer_commands.json

```diff
@@ -7188,7 +7188,7 @@
     },
     {
       "description": "Manipulate discovered rules.",
-      "name": "discovery-rule",
+      "name": "discovery_rule",
```

Comment:
think we can insert that without any consequences (for the current implementation), any way running  the test to verify (it takes a long time)